sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -821,6 +821,7 @@ object SparkSession extends Logging {
      *
      * @since 2.0.0
      */
+    @deprecated("use conf", "3.3.1")
```

Comment:
We wouldn't introduce this change in 3.3.1, I think. 3.4.0 at best. But why do we need to deprecate these? won't most invocations bind to these anyway and then users get deprecation warnings?

Reply:
@srowen 
thx for your comment.

I will change `3.4.0`.
I think, `config` method deprecated is better than or not.
because same method `conf`.
conf method is extended from `config`.

but, you don't like, I will remove `deprecated`
can you give me answer?

Reply:
Right. But then there just isn't much point in this change. I'd leave it all as is